src/types/model.ts

```diff
@@ -58,7 +58,8 @@ export type ModelsOptions = OptionalValues<
  * @interface
  */
 export type ModelSpec<S extends SyncFuncConstraint> = {
-  name: string;
+  // LEGACY: we're phasing out the use of name in favour of channelName
+  name?: string;
```

Comment:
Let's just remove this entirely - what else depends on it that's not updated in this PR?

Reply:
That's a breaking change. If someone uses our SDK already, then their linter would go crazy and they'd need to refactor all their instances when updating. we're okay with that?

Reply:
Yes I'm happy to remove this, we will just bump the release tag according to semver :)